plugins/koterm.koplugin/profile

```diff
@@ -0,0 +1,27 @@
+# This file gets executed on every start of the KOTerm shell.
+# Do not edit this file!
+# You may change the contents of 'aliases' and 'profile.user'
+#
+
+export PATH="$PATH:$(pwd)/plugins/koterm.koplugin/scripts"
+export TERM="vt52"
+
+PS1="$ "
+if [ $(id -u) -eq 0 ]; then
+    PS1="# "
+fi
+export PS1
+
+# Source a user profile if it exists.
+USER_PROFILE="$(pwd)/plugins/koterm.koplugin/scripts/profile.user"
+if [ -f "$USER_PROFILE" ]; then
+    source "$USER_PROFILE"
```

Comment:
source is a bashism, probably not quite safe, see checkbashisms (but shellcheck will probably also mention it)